.github/workflows/dockerhub-description.yml

```diff
@@ -0,0 +1,26 @@
+name: Update Docker Hub Description
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - README.md
+      - .github/workflows/dockerhub-description.yml
+jobs:
+  dockerHubDescription:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Modify readme for DockerHub
+        run: |
+          sed -i '1,2d' README.md
+          sed -i 's/(LICENSE)/(https:\/\/github.com\/nginxinc\/nginx-prometheus-exporter\/blob\/master\/LICENSE)/' README.md
```

Comment:
why the extra 'sed' here ? The other PR only has onehttps://github.com/nginxinc/nginx-ingress-operator/pull/108/files

Reply:
it's the only repo where we have the LICENSE linked in the README, but mainly it was just a "test" to see what we can do if we want to customize them more.